src/models/ReproConfig.ts

```diff
@@ -12,8 +12,10 @@ export interface ReproConfig {
    * Repro steps mode for Crash Reporting.
    *
    * @default ReproStepsMode.enabledWithNoScreenshots
+   *
+   * When this is set, `anr`, `appHangs`,`fatalCrash`,`nonFatalCrash`,`forceRestart`, and `outOfMemory` will be ignored.
    */
-  crash?: ReproStepsMode;
+  allCrashes?: ReproStepsMode;
```

Comment:
We can't remove the old `crash` name as that would be a breaking change, we can either keep it `crash` as is or keep it with a `@deprecated` flag and add the new `allCrashes` one then remove `@deprecated` in a future major release.